src/assets/styles/pikaday.scss

```diff
@@ -0,0 +1,247 @@
+/*!
```

Comment:
We could also keep this file true to the original and add overrides to a second file called something like `pikaday-overrides`. It guess it depends on how much we wanna change...